athena-neptune/docs/aws-glue-sample-scripts/sample-cli-script.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
```

Comment:
please add comments that describe the purpose of this script.

Reply:
This is a sample script to create Glue catalog and tables for sample air routes dataset